packages/server/graphql/private/mutations/endTrial.ts

```diff
@@ -0,0 +1,38 @@
+import getRethink from '../../../database/rethinkDriver'
+import getKysely from '../../../postgres/getKysely'
+import setTierForOrgUsers from '../../../utils/setTierForOrgUsers'
+import setUserTierForOrgId from '../../../utils/setUserTierForOrgId'
+import {MutationResolvers} from '../resolverTypes'
+
+const endTrial: MutationResolvers['endTrial'] = async (_source, {orgId}, {dataLoader}) => {
+  const now = new Date()
+  const r = await getRethink()
+  const pg = getKysely()
+
+  const organization = await dataLoader.get('organizations').load(orgId)
+
+  // VALIDATION
+  if (!organization.trialStartDate) {
+    throw new Error('No trial active for org')
```

Comment:
-1 You defined an error payload, so you should return an error here instead of throwing

Reply:
Fixed